icu4j/main/classes/core/src/com/ibm/icu/number/FormattedNumber.java

```diff
@@ -136,6 +138,51 @@ public MeasureUnit getOutputUnit() {
         return this.outputUnit;
     }
 
+    /**
+     * Gets the noun class of the formatted output. Returns `NOT_SUPPORTED` when the noun class
+     * is not supported yet.
+     *
+     * @throws UResourceTypeMismatchException
+     * @return `NounClass`
+     * @draft ICU 71.
```

Comment:
`@provisional` as well (search the code base for an example)

Reply:
I have searched and I did not found any example, are you sure it is still needed?

Reply:
Oh; I guess we don't have them anymore! #1616

Reply:
great the, we are done !